externals/kyuubi-spark-sql-engine/pom.xml

```diff
@@ -224,6 +224,7 @@
                             <include>org.apache.curator:curator-recipes</include>
                             <include>org.apache.hive:hive-service-rpc</include>
                             <include>org.apache.thrift:*</include>
+                            <include>org.apache.logging.log4j:*</include>
```

Comment:
Do we need to package log4j?

Reply:
without log4j, we get a ClassNotFoundException
```
Exception in thread "main" java.lang.NoClassDefFoundError: org/apache/logging/log4j/Level
        at java.lang.Class.getDeclaredMethods0(Native Method)
        at java.lang.Class.privateGetDeclaredMethods(Class.java:2701)
        at java.lang.Class.privateGetPublicMethods(Class.java:2902)
        at java.lang.Class.privateGetPublicMethods(Class.java:2911)
        at java.lang.Class.getMethods(Class.java:1615)
        at scala.Enumeration.populateNameMap(Enumeration.scala:193)
        at scala.Enumeration.$anonfun$nameOf$1(Enumeration.scala:212)
Caused by: java.lang.ClassNotFoundException: org.apache.logging.log4j.Level
	at java.net.URLClassLoader.findClass(URLClassLoader.java:382)
	at java.lang.ClassLoader.loadClass(ClassLoader.java:424)
	at java.lang.ClassLoader.loadClass(ClassLoader.java:357)
```

Reply:
it should not happen, can you provide a reproducable steps?

Reply:
ok，remove this line of code，build and run ut such as  `org.apache.kyuubi.operation.PlanOnlyOperationSuite`，we will get this error. if the warn string in `OperationModes`  is not import，maybe we can remove it？

Reply:
> build and run ut such as ...

Could you elaborate more about it? e.g. using JDK 11 on macOS, run command "build/mvn xxxx" could cause xxx error

Reply:
ok, using JDK 8 on Ubuntu 20, run commond `build/mvn install -Pspark-3.2 -pl dev/kyuubi-codecov -am -Dtest=none -DwildcardSuites=org.apache.kyuubi.operation.PlanOnlyOperationSuite` could cause `ClassNotFoundException: org.apache.logging.log4j.Level`